addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -199,10 +199,16 @@
                                         </td>
                                         <td class='text-center td-wish-btn align-middle'>
                                             <input name="product_id" t-att-value="wish.product_id.id" type="hidden"/>
-                                            <a t-if="combination_info['prevent_zero_price_sale']" t-att-href="website.contact_us_button_url" class="btn btn-primary btn_cta">Contact Us</a>
-                                            <button id="add_to_cart_button" t-else="" type="button" role="button" class="btn btn-secondary btn-block o_wish_add mb4" >
-                                                Add <span class='d-none d-md-inline'>to Cart</span>
-                                            </button>
+                                            <div t-if="combination_info['prevent_zero_price_sale']">
+                                                <a t-att-href="website.contact_us_button_url"
+                                                   class="btn btn-primary btn_cta">Contact Us
+                                                </a>
+                                            </div>
+                                            <div t-else="">
+                                                <button id="add_to_cart_button" type="button" role="button"
+                                                        class="btn btn-secondary btn-block o_wish_add mb4">Add<span class='d-none d-md-inline'>to Cart</span>
+                                                </button>
+                                            </div>
```

Comment:
Why adding a parent `<div>` ?

Reply:
it seemed that in order to make t-if and t-else work, there was a need for both of them to be divs. before making it a div, t-else did not work.